cli/flags.rs

```diff
@@ -79,6 +82,13 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
         .short("r")
         .long("reload")
         .help("Reload source code cache (recompile TypeScript)"),
+    ).arg(
+      Arg::with_name("config")
+        .short("c")
+        .long("config")
+        .value_name("FILE")
+        .help("Load compiler configuration file")
+        .takes_value(true),
```

Comment:
Just feed for thought here: If we ever find a need to introduce deno specific config, we'll have trouble with appropriate flag name, how about `--ts-config` for TS compiler configuration?

Reply:
Personal opinion, if we ever need a custom config file for Deno, we are probably doing it wrong.  Even if we allowed setting of flags via a configuration file, I would hope we would expand the syntax of the TypeScript format, where `"compilerOptions"` would still be provided to the compiler and the rest parsed by Deno.  In my opinion, Deno and TypeScript should always be synergistic instead of some sort of "adjunct".

Reply:
> Deno and TypeScript should always be synergistic instead of some sort of "adjunct".

Fits the Philosphy. Thumb up.

Reply:
@kitsonk thanks for response, I generally agree with your opinion - now looking at #1739 it makes a lot of sense to make it single `--config` that other compilers can use.